src/main/java/carpet/logging/logHelpers/TNTLogHelper.java

```diff
@@ -24,18 +26,30 @@ public void onPrimed(double x, double y, double z, Vec3d motion)
     /**
      * Runs when the TNT explodes. Expects the position of the TNT.
      */
-    public void onExploded(double x, double y, double z)
+    public void onExploded(double x, double y, double z, long gametime)
     {
+        if (!(lastGametime == gametime)){
+            tntCount = 0;
+            lastGametime = gametime;
+        }
+        tntCount++;
         LoggerRegistry.getLogger("tnt").log( (option) -> {
             switch (option)
             {
                 case "brief":
                     return new BaseText[]{Messenger.c(
+                            "g  #" + String.valueOf(tntCount),
```

Comment:
yeah, if you could just remove this one line from brief that would be better imo